index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"> <!-- Bootstrap-->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"> <!-- Icons-->
+  <link rel="stylesheet" href="styles.css">
+  <title>Malcom's Capstone </title>
+</head>
+<body>
+  <!-- Modal div -->
+  <div class="modalcpst display-none">
+    <div class="modal__exit-button container">
+        <i class="material-icons" id="exit-button">close</i>
+    </div>
+    <nav class="modal__navigation">
+        <ul class="container">
+            <li class="modal__navigation__li"><a href="#program">Main Program</a></li>
+            <li class="modal__navigation__li"><a href="#speakers">Speakers</a></li>
+            <li class="modal__navigation__li"><a href="about.html">About Us</a></li>
+            <li class="modal__navigation__li"><a href="#footer">Partners</a></li>
+        </ul>
+    </nav>
+  </div>
+
+  <!-- Topdiv Section -->
+  <section class="topPageDiv">
+    <div class="col-12 col-md-8"></div>
+    <div class="col-6 col-md-4">
+      <img src="./elements/facebook.png" class="topPage__img" alt="Facebook logo">
+      <img src="./elements/twitter.png" class="topPage__img" alt="Twitter logo">
+      <span class="topPage__text">English</span>
+      <span class="topPage__text">My Page</span>
+      <span class="topPage__text">Logout</span>
+    </div>
+  </section>
+
+  <!-- Home Section -->
+  <section id="home">
+    <header>
+      <div class="hamburger" id="hamburger-menu">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+      </div>
+      <nav class="navbar navbar-expand-sm bg-light navbar-light">
+        <div class="col-12 col-md-1"></div>
+        <div class="container-fluid">
+          <a class="navbar-brand" href="index.html">
+            <img src="./elements/logo.png" alt="Logo image" class="d-inline-block align-top logo"> 
+          </a>
+        </div>
+        <div class="container">
+          <ul class="navbar-nav">
+            <li class="nav-item"><a href="#home" class="nav-link">Home</a></li>
+            <li class="nav-item"><a href="#speakers" class="nav-link">Speakers</a></li>
+            <li class="nav-item"><a href="#program" class="nav-link">Program</a></li>
+            <li class="nav-item"><a href="about.html" class="nav-link">About Us</a></li>
+            <li class="nav-item"><a href="#contact" class="nav-link">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="main-text">
+        <h1 class="main-text__title">Cosmopolitan Affirming Community <br> Global Summit <br> 2023</h1>
+        <p class="main-text__bio">
```

Comment:
- [ ] While you did a good job creating all the elements inside the home page :+1:, you should also add the text that is above the main title.
- [ ] Kindly, it is required to use the same colors from the template in each element. I've noticed that your title has a different color.

**Your design**

![Screenshot from 2023-02-09 06-22-41](https://user-images.githubusercontent.com/105117832/217800187-6637bede-2a39-4052-a745-dbdd72c12c43.png)

**Template**

![Screenshot from 2023-02-09 06-23-03](https://user-images.githubusercontent.com/105117832/217800281-0ba1e28c-c567-43f4-a195-be47b7070385.png)

Reply:
@IngDavidTM Thank you for your timely and well-detailed review, I appreciate it. However, I have a few questions like in this section you highlight the use of a different colour. In the project requirements, we are allowed room to use different images thus my concern why flag this?

Reply:
Hi @Malcom-Charlie, yes you can change the images that you need inside your HTML, but the background should be the same. I know that is confusing, but microverse asks us to review the background also.

![Screenshot from 2023-02-09 09-44-54](https://user-images.githubusercontent.com/105117832/217845278-e59e4cbe-0241-4c10-b054-b5634ef78c91.png)